focus-group.js

```diff
@@ -1,6 +1,12 @@
-export function menuKeyUX(options) {
+export function focusGroupKeyUX(options) {
   return window => {
-    let inMenu = false
+    const supportedRoles = {
```

Comment:
I mean

```js
const ROLES = {
  'menuitem': ["menu", "menubar"],
  'option': ["listbox"],
  'tab': ["tablist"]
}

export function focusGroupKeyUX(options) {
```

Since we do not change the list, we do not need to re-create it on every key press.